Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/FareCalculator.hs

```diff
@@ -260,10 +259,10 @@ calculateFareParameters params = do
                 ConstantPlatformFee charge -> fromIntegral charge
           roundToIntegral (baseFee + baseFee * platformFeeInfo'.cgst + baseFee * platformFeeInfo'.sgst)
 
-countFullFareOfParamsDetails :: DFParams.FareParametersDetails -> (Money, Money, Money)
+countFullFareOfParamsDetails :: DFParams.FareParametersDetails -> (Money, Money)
 countFullFareOfParamsDetails = \case
-  DFParams.ProgressiveDetails det -> (fromMaybe 0 det.extraKmFare, det.deadKmFare, 0) -- (partOfNightShiftCharge, notPartOfNightShiftCharge)
-  DFParams.SlabDetails det -> (0, 0, fromMaybe 0 det.platformFee)
+  DFParams.ProgressiveDetails det -> (fromMaybe 0 det.extraKmFare, det.deadKmFare) -- (partOfNightShiftCharge, notPartOfNightShiftCharge)
+  DFParams.SlabDetails det -> (0, fromMaybe 0 det.platformFee)
```

Comment:
If we will make it part of `notPartOfNightShiftCharge` then `govtCharges` will also add on it which should not be the case. We should not add `govtCharges` on platfromFee as platformFee has it own CGST and sgst.